src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -1527,20 +1529,22 @@ void InsertSplitAligningFilterPass::run() {
 
                     auto inputData = splitOutput;
 
-                    size_t aligned64_offset = std::max(0, static_cast<int>(ALIGN64(currentOffset) - 64));
+                    size_t aligned_offset = std::max(
```

Comment:
whole code 
`std::max(...`  is used two times in gna_pass_manager it could be common function in gra_pass_manager.cpp if this is only gna_pass_manager specific